lib/_http_agent.js

```diff
@@ -91,7 +91,7 @@ Agent.defaultMaxSockets = Infinity;
 Agent.prototype.createConnection = net.createConnection;
 
 // Get the key for a given set of request options
-Agent.prototype.getName = function(options) {
+Agent.prototype.getName = function getName(options) {
```

Comment:
Generally in flamegraphs you see fileName & function name, the two together help a bunch.

In a heapsnapshot you only see function name, however a heap snapshot does show whom holds a reference to the function, if a function is referenced from a class then that anchor point makes it easy to find.

I think having any kind of name at all is great. Having a unique name like `Agent_getName` does improve it slightly but that might get really annoying for coding style / redundency.